bugbug/utils.py

```diff
@@ -245,14 +245,28 @@ def zstd_compress(path: str) -> None:
     if not os.path.exists(path):
         raise FileNotFoundError(errno.ENOENT, os.strerror(errno.ENOENT), path)
 
-    subprocess.run(["zstdmt", "-f", path], check=True)
+    try:
+        subprocess.run(["zstdmt", "-f", path], check=True)
+    except FileNotFoundError:
+        # Use zstandard API for compression if zstdmt is not available
+        cctx = zstandard.ZstdCompressor()
+        with open(path, "rb") as input_f:
+            with open(f"{path}.zst", "wb") as output_f:
+                cctx.copy_stream(input_f, output_f)
 
 
 def zstd_decompress(path: str) -> None:
     if not os.path.exists(f"{path}.zst"):
         raise FileNotFoundError(errno.ENOENT, os.strerror(errno.ENOENT), path)
 
-    subprocess.run(["zstdmt", "-df", f"{path}.zst"], check=True)
+    try:
+        subprocess.run(["zstdmt", "-df", f"{path}.zst"], check=True)
+    except FileNotFoundError:
+        # Use zstandard API for decompression if zstdmt is not available
```

Comment:
Usually, code comments that explain "why" are more useful than the ones that explain "what". When you feel that you need a comment to explain what the code is doing, in most cases the solution is to simplify the code instead of adding a comment.

```suggestion
       # Fallback to use zstandard API when zstdmt is not available to make life
       # easier for new contributors.
```

In this context, I am keen to add a warning log instead of a comment.